modules/git/repo_compare.go

```diff
@@ -259,3 +263,31 @@ func (repo *Repository) GetDiffFromMergeBase(base, head string, w io.Writer) err
 	}
 	return err
 }
+
+// ReadPatch will check if a diff patch exists and return stats
+func (repo *Repository) ReadPatch(prID int64) (commits string, patch []int) {
```

Comment:
This function is still very in early stage and I was wondering if I should just grab the first commit of a patch file, or read also the git apply --stat